tests/ffi/modules/test.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -euvx
+
+# shellcheck disable=SC1091
+
+. ../common/prepare.sh
+
+export QM_HOST_REGISTRY_DIR="/var/qm/lib/containers/registry"
+export QM_REGISTRY_DIR="/var/lib/containers/registry"
+
+disk_cleanup
+prepare_test
+reload_config
+
+exec_cmd "podman run -d --rm --replace -d --name ffi-asil \
+          quay.io/centos-sig-automotive/ffi-tools:latest  > /dev/null"
+
+image_id=$(podman images | grep quay.io/centos-sig-automotive/ffi-tools | awk -F " " '{print $3}')
```

Comment:
the same question I did in PR #448. Could you please check there ?

Reply:
+2 here from also

Reply:
Added if statement.